custom_components/enphase_envoy_custom/translations/nl.json

```diff
@@ -13,12 +13,14 @@
         "step": {
             "user": {
                 "data": {
-                    "host": "Host",
+                    "host": "Envoy adres (naam of ip)",
                     "password": "Wachtwoord",
-                    "username": "Gebruikersnaam"
+                    "username": "Gebruikersnaam",
+                    "serial": "Envoy serie nummer",
+                    "use_enlighten": "Gebruik Enphase Token"
                 },
-                "description": "Voer voor nieuwere modellen gebruikersnaam 'envoy' in zonder wachtwoord. Voer voor oudere modellen gebruikersnaam `installer` in zonder wachtwoord. Voer voor alle andere modellen een geldige gebruikersnaam en wachtwoord in."
+                "description": "* Voor oudere modellen vul gebruikersnaam `installer` in zonder wachtwoord. \n* Voor nieuwere modellen vul gebruikersnaam 'envoy' in zonder wachtwoord. \n* Voor firmware D7.0.0 en hoger schakel `gebruik enphase token` in en gebruik Enphase website gebruikersnaam en wachtwoord en vul envoy serienummer in. \n* Voor alle andere modellen voer een geldige gebruikersnaam en wachtwoord in."
```

Comment:
This completely changes the text compared to the English version, so I wouldn't consider this a translation update actually.

Reply:
Agree, my bad as I had changed en.json locally too to make options clearer and  only committed the nl.json. Guess it would be a separate PR and review on the subject of the description clarity and change this one to the original line with the D7 firmware remark added the same way as in en.json. I'll separate both.

Reply:
chenged in next commit
![Image-1343](https://user-images.githubusercontent.com/120491684/230658833-228fc969-859d-4682-9e07-a8f85cd5f6fa.jpg)